core/src/main/java/org/jdbi/v3/core/enums/DatabaseValue.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.enums;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.jdbi.v3.core.qualifier.Qualifier;
+import org.jdbi.v3.meta.Beta;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+@Qualifier
```

Comment:
This shouldn't be a qualifier annotation, should it?

Reply:
I don’t know what `@Qualifier` does, it’s an oversight from copy & paste.

Reply:
Qualifying annotations are a recent addition to Jdbi, typically used to distinguish when multiple SQL data types resolve to the same Java type. e.g. both `VARCHAR` and `NVARCHAR` SQL columns are represented as `String` in Java.

http://jdbi.org/#_qualified_types

Reply:
https://github.com/jdbi/jdbi/commit/c6e243cc4c374fe3510b3b3a967bc4c8f6e31cdc